dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -190,7 +190,7 @@ interface CompanyApi {
     /**
      * A method to update company informtion for one specific company identified by its company Id
      * @param companyId identifier of the company in dataland
-     * @param companyInformation includes the company information
+     * @param companyInformationPatch includes the company information
```

Comment:
does this renaming have a specific reason?

Reply:
@fsoffel maybe you know more about the change? But if i had to gues it was to make it clearer what the input parameter is about

Reply:
No renaming happened. I just fixed the javadocs.